cms/admin/permissionadmin.py

```diff
@@ -51,8 +51,9 @@ def get_queryset(self, request):
         # can see only permissions for users which are under him in tree
 
         # here an exception can be thrown
+        model = self.model
```

Comment:
Ok for settingsadmin.py:42
I can't find permissionadmin.py:54, permissionadmin.py:117: aren't those in this PR?
Use the ``qs = self.model...`` syntax everywhere in this PR, we will change settingsadmin later